packages/ove-app-networks/README.md

```diff
@@ -0,0 +1,57 @@
+# Networks App
+
+This app supports visualisation of network diagrams using the OVE framework. It is based on [Sigma](http://sigmajs.org/), a JavaScript library dedicated to drawing graphs. This supports datasets specified in Sigma's JSON format or [GEXF](https://gephi.org/gexf/format/). Sigma supports Canvas, SVG and WebGL rendering.
+
+## Application State
+
+The state of this app has a format similar to what is provided below:
+
+```json
+{
+    "jsonURL": "/data/sample.json",
+    "settings": {
+        "autoRescale": true,
+        "clone": false,
+        "defaultNodeColor": "#ec5148"
+    },
+    "renderer": "canvas"
+}
+```
+
+The `jsonURL` field should be replaced with with `gexfURL` field depending on the format in which the dataset is specified. Information on all available `settings` can be found in [Sigma documentation](https://github.com/jacomyal/sigma.js/wiki/Settings). The `renderer` field is optional and defaults to `webgl`.
```

Comment:
IIRC correctly, after we converted the CyberSparks networks they didn't render correctly, and the problem was due to a bug in Sigma.js' WebGL rendering so we reverted to using it's canvas rendered instead.

Did this bug get fixed? If not, we probably shouldn't default to `webgl`.

Reply:
We have not resolved that bug, but that is specifically down to the settings and the attributes passed in that app. This is not in scope for OVE, but a issue that must be reported on the repository where that app exists. I can also confirm there is no bug in Sigma when it comes to WebGL rendering, this has been tested using other examples of theirs.